src/lib/modules/Document/utils.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { Truncate } from '@components/Truncate';
+import find from 'lodash/find';
+import _isEmpty from 'lodash/isEmpty';
+
+export const renderSubtitle = (alternativeTitles) => {
+  if (_isEmpty(alternativeTitles)) {
```

Comment:
If I am not mistaken, `lodash` is already checking for empty/null values, so this is not needed.